README.rst

```diff
@@ -162,27 +161,35 @@ If TensorBoard is installed, automatically visualize all trial results:
 RLlib Quick Start
 -----------------
 
-.. image:: https://github.com/ray-project/ray/raw/master/doc/source/images/rllib-wide.jpg
+.. image:: https://github.com/ray-project/ray/raw/master/doc/source/images/rllib/rllib-logo.svg
 
-`RLlib`_ is an open-source library for reinforcement learning built on top of Ray that offers both high scalability and a unified API for a variety of applications.
+`RLlib`_ is an industry-grade, open-source library for reinforcement learning (RL) built on top of Ray.
+RLlib offers high scalability and unified APIs for a variety of industry- and research applications.
 
 .. code-block:: bash
 
-  pip install tensorflow  # or tensorflow-gpu
-  pip install "ray[rllib]"
+    $ pip install "ray[rllib]" tensorflow  # or torch
+
+
+.. .. literalinclude:: rllib/examples/documentation/rllib_on_ray_readme.py
```

Comment:
This did not seem to work. I'm not sure why. Getting a sphinx error, which I don't get when doing this exact same thing inside the `ray/doc/source` folder.

Reply:
It doesn't work because README.rst is not compiled

Reply:
Makes sense. github just displays as is.
Ok, we have to leave the explicit code as-is then (non shared).

Reply:
Created two slightly different examples for the ray README as well as for the RLlib README. Copies of these are being CI-tested.